test/test.hsd

```diff
@@ -8,46 +8,40 @@ Geometry {
   }
 }
 Driver {}
-Hamiltonian {
-  DFTB {
-    Scc = Yes
-    SccTolerance = 1e-10
-    MaxSccIterations = 1000
-    Mixer {
-      Broyden {}
-    }
-    MaxAngularMomentum {
-      H = SelectedShells {"s" "s" }
-      O = SelectedShells {"s" "p" }
-    }
-    Dispersion = SlaterKirkwood {
-      PolarRadiusCharge [AA^3,AA,] = {
-        1.030000    3.800000    2.820000
-      }
-    }
-    Filling {
-      Fermi {
-        Temperature [Kelvin] = 1e-08
-      }
+Hamiltonian = DFTB {
```

Comment:
Is it worth adding comment marks to the input example, a) just in case it breaks something in future, and b) shows that happens on parsing?